### PR TITLE
addresscount

### PR DESCRIPTION
**Pull request by zhiyuan2007**
#785

### CODE REVIEW

File: src/gui/explorer.go [hunk 1]
```diff
@@ -244,6 +246,25 @@ func getRichlist(gateway *daemon.Gateway) http.HandlerFunc {
...
+func getAddressCount(gateway *daemon.Gateway) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet {
+			wh.Error405(w)
+			return
+		}
+
+		addrCount, err := gateway.GetAddressCount()
+		if err != nil {
+			wh.Error400(w, "internal error when get addresscount")
```
Comment (gz-c) [inline src/gui/explorer.go]: Error500

**Comment by zhiyuan2007** [comment]
merged richlist

**Comment by gz-c** [comment]
merge conflict

**Comment by zhiyuan2007** [comment]
fixed

**Pull request merged**